core/templates/newspapers.html

```diff
@@ -25,6 +25,7 @@
         <thead>
         <tr>
             <th class="headerSortDown left_no_border">State</th>
```

Comment:
I am not sure it would when the page first appeared, but I haven't looked at the javascript at all.  It's my impression that you might get something like:

Lincoln, Lancaster, NE
Albany, County, NY
Cody, County, WY

When you first went to the page, which would look out of order, when in fact it would have been sorted by state.  Otherwise, we could make the entire view return the results with sort by titles.all().order_by("place_of_publication") or something like that, I suppose.  But do we want to do that in this sprint or should we add an issue?

I like the idea of using a setting for the location type, but I don't know that it will be a super fast fix.